lib/private/AppFramework/Utility/ControllerMethodReflector.php

```diff
@@ -33,12 +33,15 @@
  */
 namespace OC\AppFramework\Utility;
 
+use Attribute;
 use OCP\AppFramework\Utility\IControllerMethodReflector;
 
 /**
  * Reads and parses annotations from doc comments
  */
 class ControllerMethodReflector implements IControllerMethodReflector {
+
+	protected ?\ReflectionMethod $reflection;
```

Comment:
To be more specific because exist other types of reflections:

```suggestion
	protected ?\ReflectionMethod $reflectionMethod;
```